code/datums/traits/negative.dm

```diff
@@ -36,21 +36,72 @@
-		if("Security Officer")
-			heirloom_type = /obj/item/book/manual/wiki/security_space_law
-		if("Scientist")
-			heirloom_type = /obj/item/toy/plush/slimeplushie
-		if("Assistant")
-			heirloom_type = /obj/item/storage/toolbox/mechanical/old/heirloom
+
+	if(is_species(H, /datum/species/moth))
+		heirloom_type = /obj/item/flashlight/lantern/heirloom_moth
+	else
+		switch(quirk_holder.mind.assigned_role)
+			//Service jobs
+			if("Clown")
+				heirloom_type = /obj/item/bikehorn/golden
+			if("Mime")
+				heirloom_type = /obj/item/reagent_containers/food/snacks/baguette
+			if("Janitor")
+				heirloom_type = pick(/obj/item/mop, /obj/item/caution, /obj/item/reagent_containers/glass/bucket)
+			if("Cook")
+				heirloom_type = pick(/obj/item/reagent_containers/food/condiment/saltshaker, /obj/item/kitchen/rollingpin, /obj/item/clothing/head/chefhat)
+			if("Botanist")
+				heirloom_type = pick(/obj/item/cultivator, /obj/item/reagent_containers/glass/bucket)
+			if("Bartender")
+				heirloom_type = pick(/obj/item/reagent_containers/glass/rag, /obj/item/clothing/head/that, /obj/item/reagent_containers/food/drinks/shaker)
+			if("Curator")
+				heirloom_type = pick(/obj/item/pen/fountain, /obj/item/storage/pill_bottle/dice)
+			if("Assistant")
+				heirloom_type = /obj/item/storage/toolbox/mechanical/old/heirloom
+			//Security/Command
+			if("Captain")
+				heirloom_type = /obj/item/reagent_containers/food/drinks/flask/gold
+			if("Head of Security")
+				heirloom_type = /obj/item/book/manual/wiki/security_space_law
+			if("Warden")
+				heirloom_type = /obj/item/book/manual/wiki/security_space_law
+			if("Security Officer")
+				heirloom_type = pick(/obj/item/book/manual/wiki/security_space_law, /obj/item/clothing/head/beret/sec)
+			if("Detective")
+				heirloom_type = /obj/item/reagent_containers/food/drinks/bottle/whiskey
+			if("Lawyer")
+				heirloom_type = pick(/obj/item/gavelhammer, /obj/item/book/manual/wiki/security_space_law)
+			//RnD
+			if("Research Director")
+				heirloom_type = /obj/item/toy/plush/slimeplushie
+			if("Scientist")
+				heirloom_type = /obj/item/toy/plush/slimeplushie
+			if("Roboticist")
+				heirloom_type = pick(subtypesof(/obj/item/toy/prize)) //nerd
+			//Medical
+			if("Chief Medical Officer")
+				heirloom_type = pick(/obj/item/clothing/neck/stethoscope, /obj/item/bodybag)
+			if("Medical Doctor")
+				heirloom_type = pick(/obj/item/clothing/neck/stethoscope, /obj/item/bodybag)
+			if("Chemist")
+				heirloom_type = /obj/item/book/manual/wiki/chemistry
+			if("Virologist")
+				heirloom_type = /obj/item/reagent_containers/syringe
+			//Engineering
+			if("Chief Engineer")
+				heirloom_type = pick(/obj/item/clothing/head/hardhat/white, /obj/item/airlock_painter)
+			if("Station Engineer")
+				heirloom_type = pick(/obj/item/clothing/head/hardhat, /obj/item/airlock_painter)
+			if("Atmospheric Technician")
+				heirloom_type = pick(/obj/item/lighter, /obj/item/lighter/greyscale, /obj/item/storage/box/matches)
```

Comment:
The matchbox is amazing here.

It's probably safe to give engineering jobs a analyzer, crowbar, screwdriver, welder, wirecutter, or wrench since they start with a filled toolbelt.

It actually makes lategame more interesting because it contrasts with the incentive to replace all tools if the advanced tools nodes come ready.